service/controller/orgpermissions/orgpermissions.go

```diff
@@ -0,0 +1,61 @@
+package orgpermissions
+
+import (
+	"github.com/giantswarm/k8sclient/v4/pkg/k8sclient"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/operatorkit/v2/pkg/controller"
+	"github.com/giantswarm/operatorkit/v2/pkg/resource"
+	rbacv1 "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"github.com/giantswarm/rbac-operator/pkg/project"
+)
+
+type OrgPermissionsConfig struct {
+	K8sClient k8sclient.Interface
+	Logger    micrologger.Logger
+}
+
+type OrgPermissions struct {
+	*controller.Controller
+}
+
+func NewOrgPermissions(config OrgPermissionsConfig) (*OrgPermissions, error) {
+	var err error
+
+	var resources []resource.Interface
+	{
+		c := orgPermissionsResourcesConfig(config)
+
+		resources, err = NewOrgPermissionsResources(c)
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
+	}
+
+	var organizationPermissionsController *controller.Controller
+	{
+		c := controller.Config{
+			K8sClient: config.K8sClient,
+			Logger:    config.Logger,
+			NewRuntimeObjectFunc: func() runtime.Object {
+				return new(rbacv1.RoleBinding)
```

Comment:
What if organization doesn't have any rolebindings?

Reply:
There is at least 1 rolebinding for customer admin group.
But also it is fine if there are no other rolebindings. It is just something customer admin decides on its own if any group/user needs access

Reply:
But if there are no rolebindings this controller will do nothing meaning that there won't be a role for reading the org.

IMO this should be reconciling org CR, or the handler moved to the other controller.

Reply:
We talked with @corest , I read it wrong thinking that subjects for the new role binding come from nowhere...